library/src/lib/icon/icon.component.ts

```diff
@@ -14,23 +14,29 @@ const BASE_ICON_CLASS = 'sap-icon';
 const PREFIX_ICON_CLASS = BASE_ICON_CLASS + '--';
 
 /**
- * The directive that represents an icon. 
+ * The Component that represents an icon.
  *
  * ```html
  * <fd-icon [glyph]="cart-approval" [size]="'l'"></fd-icon>
- * ```
+ // * ```
+ // * Or
+ // * ```html
+ // * <fd-icon [size]="'l'">cart-approval</fd-icon>
```

Comment:
This actually displays in the docs sortof. I'm guessing you commented it out because it's not currently working? We should envision implementing this in the future

![image](https://user-images.githubusercontent.com/24924530/60347671-e56adb00-998b-11e9-97af-87617fcd3c02.png)